### PR TITLE
FIX simple case of multientity with same entity_names and different t…

### PR DESCRIPTION
**Pull request by mrutid**
FIX simple case of multientity with same entity_names and different types.

Some problems maybe found with same measure or attribute names. More cases will follow

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -8,3 +8,4 @@
 - Add: new JEXL transformation for Array.slice(init, end) in order to simplify binary-frame-string field extraction.
 - Fix: amend null processing for NGSI-LD (#1118)
 - Updated JEXL dependecy from 2.1.1 to 2.3.0 to allow the usage of NGSI operators ($inc, etc.)
+- Support of same entity_id and diferent entity_type in multientity plugin
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:11]: Is there some associated issue? Maybe https://github.com/telefonicaid/iotagent-node-lib/issues/1127 ?

Comment (mrutid) [reply CHANGES_NEXT_RELEASE:11]: yes... but it doesn't fix the more general scenario.

File: package.json [hunk 2]
```diff
@@ -38,6 +38,7 @@
     "prettier:text": "prettier 'README.md' 'doc/*.md' 'doc/**/*.md' --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
     "test": "nyc --reporter=text mocha --recursive 'test/**/*.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
     "test:expression": "nyc --reporter=text mocha --recursive 'test/unit/expressions/*.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
+    "test:mulientity": "nyc --reporter=text mocha --recursive 'test/unit/ngsiv2/plugins/multientity-plugin_test.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
```
Comment (fgalan) [inline package.json:41]:
```suggestion
    "test:multientity": "nyc --reporter=text mocha --recursive 'test/unit/ngsiv2/plugins/multientity-plugin_test.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
```

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: issue #1127

**Review by AlvaroVega — APPROVED** [review]
LGTM

File: package.json (hunk 2 above)
Comment (fgalan) [reply package.json]: Fixed in c20b733

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by mrutid** [comment]
Added a new test case with shared attr names and shared measures. Fake Object_ids and expression should be used at configuration.

**Comment by fgalan** [comment]
After merging PR #1134 into this branch, a conflict in  test/unit/ngsiv2/plugins/multientity-plugin_test.js  has arisen. It needs to be solved before merging in master.

File: test/unit/ngsiv2/plugins/multientity-plugin_test.js [hunk 3]
```diff
@@ -297,20 +297,20 @@ const iotAgentConfig = {
         WrongStation: {
             commands: [],
             type: 'WrongStation',
-            expressionLanguage: 'jexl',
+            expressionLanguage: 'jexl',
```
Comment (fgalan) [inline test/unit/ngsiv2/plugins/multientity-plugin_test.js:300]:
```suggestion
            expressionLanguage: 'jexl',
```

File: test/unit/ngsiv2/plugins/multientity-plugin_test.js [hunk 4]
```diff
@@ -297,20 +297,20 @@ const iotAgentConfig = {
...
             type: 'WrongStation',
-            expressionLanguage: 'jexl',
+            expressionLanguage: 'jexl',     
             lazy: [],
             active: [
                 {
-                    object_id: 'v1',
-                    name: 'type',
+                    object_id: 'v1',     
+                   name: 'type',
```
Comment (fgalan) [inline test/unit/ngsiv2/plugins/multientity-plugin_test.js:305]:
```suggestion
                    object_id: 'v1',
                    name: 'type',
```

File: test/unit/ngsiv2/plugins/multientity-plugin_test.js [hunk 5]
```diff
@@ -297,20 +297,20 @@ const iotAgentConfig = {
...
                     type: 'string',
                     entity_name: 'WrongStation1'
                 },
                 {
                     object_id: 'v2',
                     name: 'id',
                     type: 'string',
-                    entity_name: 'WrongStation1'
+                    entity_name: 'WrongStation1'
```
Comment (fgalan) [inline test/unit/ngsiv2/plugins/multientity-plugin_test.js:313]:
```suggestion
                    entity_name: 'WrongStation1'
```

**Comment by AlvaroVega** [comment]
this branch war merged in PR https://github.com/telefonicaid/iotagent-node-lib/pull/1135

**Pull request closed without merging**